Assets/Scripts/Characters/PlayerEnergy.cs

```diff
@@ -56,7 +57,9 @@ private void HandleMovementStart(InputAction.CallbackContext context)
 
         private void ResetEnergy(InputAction.CallbackContext context)
         {
+            EnergyReset?.Invoke();
             energy = 100f;
+            _energyLabel.SetEnergy(energy);
```

Comment:
This was causing the energy amount to not update after a reset until more energy was exhausted